tests/TestCase/View/Helper/PaginatorHelperTest.php

```diff
@@ -777,13 +777,13 @@ public function testDefaultSortAndNoSort()
      */
     public function testUrlGeneration()
     {
-        $result = $this->Paginator->sort('controller');
-        $expected = [
-            'a' => ['href' => '/index?sort=controller&amp;direction=asc'],
-            'Controller',
-            '/a',
-        ];
-        $this->assertHtml($expected, $result);
+        // $result = $this->Paginator->sort('controller');
+        // $expected = [
+        //     'a' => ['href' => '/index?sort=controller&amp;direction=asc'],
+        //     'Controller',
+        //     '/a',
+        // ];
+        // $this->assertHtml($expected, $result);
```

Comment:
Should this be removed instead?

Reply:
Oops, that was temporary. Un-commented again.